elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,381 @@
+    @at-root #{&}([vertical][pfe-variant="wind"]) {
+      border-left: pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-var(surface--border);
+      text-align: left !important;
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="true"]) {
+      border-right: pfe-local(BorderWidth) pfe-var(ui--border-style) transparent;
+      border-left: pfe-local(BorderWidth) pfe-var(ui--border-style) pfe-local(highlight);
+      padding-left: calc(#{pfe-var(container-padding)} - 2px);
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]) {
+      border-right: pfe-local(BorderWidth) pfe-var(ui--border-style) transparent;
+    }
+
+    @at-root #{&}([vertical][pfe-variant="wind"][aria-selected="false"]:hover) {
+      border-right: pfe-local(BorderWidth) pfe-var(ui--border-style) transparent;
+      border-bottom: 0;
+      border-left: pfe-local(BorderWidth) pfe-var(ui--border-style) pfe-local(BorderColor--hover);
+      padding-left: calc(#{pfe-var(container-padding)} - 2px);
     }
   }
 }
 
+
+/// ===========================================================================
+/// EARTH TAB VARIANTS
+/// ===========================================================================
 :host([pfe-variant="earth"]) {
   @include browser-query(ie11) {
-    background-color: #{pfe-var(surface--lighter)};
-    color: #{pfe-var(text)};
+    background-color: pfe-fetch(surface--lighter);
+    color: pfe-fetch(text--muted);
   }
 }
 
-/// ===========================================================================
-/// WIND TAB VARIANTS / DEFAULT
-/// ===========================================================================
+:host(:not([vertical])[pfe-variant="earth"]:first-of-type) {
+  border-left-color: transparent;
+}
 
-@each $theme in $THEMES { // dark, saturated, and light
-  :host([aria-selected="true"][on="#{$theme}"]),
-  :host([on="#{$theme}"]:hover),
-  :host([pfe-variant="wind"][aria-selected="true"][on="#{$theme}"]),
-  :host([pfe-variant="wind"][on="#{$theme}"]:hover) {
-    @if $theme != "light" {
-      --pfe-tabs--Color: #{pfe-var(text--on-#{$theme})};
-    }
-    @else {
-      --pfe-tabs--Color: #{pfe-var(text)};
+:host(:not([vertical])[pfe-variant="earth"]:last-of-type) {
+  border-right-color: transparent;
+}
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][aria-selected="false"]) {
+    background-color: pfe-local(BackgroundColor--inactive); //@todo update to use opacity
+    border-color: pfe-var(surface--border);
+    border-top-width: pfe-local(BorderWidth);
+    border-top-color: transparent;
+    border-bottom-color: pfe-local(BorderColor--hover);
+    border-bottom-width: pfe-var(ui--border-width);
+  }
+
+  @at-root #{&}([pfe-variant="earth"][aria-selected="false"]:hover) {
+    border-top-color: pfe-local(BorderColor);
+  }
+
+  // nth of type is needed to override first and last of type selectors above
+  @at-root #{&}([pfe-variant="earth"][aria-selected="true"]:nth-of-type(n)) {
```

Comment:
Why override it? Should the selector above for first-of-type and last-of-type be more specific?

Reply:
its the `selected=true` that I really want, I need the first and last children of unselected tabs to not have borders. but if the tab is selected, it should have borders

Reply:
So should lines 149 and 153 have `:not([aria-selected="true"])` included in their stack instead so you're not fighting that specificity here?

Reply:
I aske because nth-of-type isn't the most efficient selector: https://www.sitepoint.com/optimizing-css-id-selectors-and-other-myths/

Reply:
good point, updated